services/QuillLMS/app/assets/stylesheets/pages/contact.scss

```diff
@@ -35,15 +40,41 @@
     font-weight: bold;
     margin-top: 20px;
   }
-  .second-paragraph {
-    margin-top: 24px;
-  }
 }
 
-.sales-container {
-  border-bottom: 1px solid #E0E0E0;
+.contact-page {
+  display: flex;
+  flex-direction: row;
+  padding: 64px 0px;
+  justify-content: center;
+  align-items: flex-start;
+  .vertical-divider {
+    width: 1px;
+    background: var(--color-grey-grey-5, #E0E0E0);
```

Comment:
Unfortunately, Figma doesn't output these in a way that's readable for our system (so the CSS is ignoring the variable and defaulting to the #E0E0E0 you specified, which is less flexible). This should be: `background: $quill-grey-5`.